src/data/finsupp/basic.lean

```diff
@@ -2295,7 +2303,16 @@ rfl
 end sum
 
 section
-variables [monoid G] [mul_action G α] [add_comm_monoid M]
+variables [has_zero M] [monoid_with_zero R] [mul_action_with_zero R M]
+
+@[simp] lemma single_one_smul (a b : α) (f : α → M) :
+  (single a (1:R) b) • (f a) = single a (f b) b :=
+by { by_cases a = b; simp [h] }
+
+end
+
+section
+variables [group G] [mul_action G α] [add_comm_monoid M]
```

Comment:
this used to be `[monoid G] [mul_action G α] [add_comm_monoid M]` - you turned it into a group, which is weird